dao/src/main/java/org/thingsboard/server/dao/audit/AuditLogServiceImpl.java

```diff
@@ -295,6 +299,10 @@ private AuditLog createAuditLogEntry(TenantId tenantId,
         futures.add(auditLogDao.saveByTenantIdAndEntityId(auditLogEntry));
         futures.add(auditLogDao.saveByTenantIdAndCustomerId(auditLogEntry));
         futures.add(auditLogDao.saveByTenantIdAndUserId(auditLogEntry));
+
+        // TODO: is this correct place to log action into sink?
```

Comment:
I think this is ok for now. Please remove TODO.

Reply:
done